data/language/en-GB.txt

```diff
@@ -4117,6 +4117,8 @@ STR_5808    :{WINDOW_COLOUR_2}Number of shops and stalls: {BLACK}{COMMA16}
 STR_5809    :{WINDOW_COLOUR_2}Number of information kiosks and other facilities: {BLACK}{COMMA16}
 STR_5810    :Disable train length limit
 STR_5811    :{SMALLFONT}{BLACK}If checked, you can have up to{NEWLINE}255 cars per train
+STR_5812    :{SMALLFONT}{BLACK}Refurbishes the ride
```

Comment:
The tip still seems a bit unclear as it's almost the exact thing as the other line
Maybe something like: 
```
STR_5812    : Rebuilds the ride to make the reliability go back to 100% for a price
```